core/vm/instructions.go

```diff
@@ -17,13 +17,13 @@
 package vm
 
 import (
+	substate "github.com/Fantom-foundation/Substate"
```

Comment:
nit: same as the comment above, let's group all 3rd party libraries together after std libraries.